salt/utils/verify.py

```diff
@@ -432,8 +444,8 @@ def check_max_open_files(opts):
         return
 
     msg = (
-        'The number of accepted minion keys({0}) should be lower than 1/4 '
-        'of the max open files soft setting({1}). '.format(
+        'In file salt-master.service: LimitNOFILE={1} should be 4 times '
+        'the number of accepted minion({0}). '.format(
```

Comment:
This is not the only way to change this.

I think the more generic log is better, because you could need to put the options into /etc/default/salt if you are using sysvinit, or another place for other init systems.

Reply:
@gtmanfred, have you created a document (or an issue) how to raise OS resource limits due to the number of accepted minions? 

I need to raise limits, again. This time for CentOS and Debian.

Thank you

Reply:
I have not, i am still waiting on more feedback in https://github.com/saltstack/salt/issues/40173